index.js

```diff
@@ -1,14 +1,264 @@
 const express = require('express');
 const connectDB = require('./config/db');
+const config = require('config');
+const nanoid = require('nanoid');
+
+
+const Url = require('./models/Url');
+const { listen } = require('express/lib/application');
 
 const app = express();
 
+
 // Connect to database
 connectDB();
 
 app.use(express.json());
 
-app.use('/', require('./routes/index'));
+// @route     POST /api/v1/shorten
+// @desc      Create short URL
+
+app.post('/api/v1/shorten', async (req, res) => {
+  try {
+    const { url, customName } = req.body;
+
+    // Validate longUrl
+    if (!url) {
+      return res.status(400).json({ error: 'Please provide a valid long URL.' });
+    }
```

Comment:
This check is better in a middleware. Please checkout [express-validator](https://www.npmjs.com/package/express-validator), it's perfect for your needs.